observelib/observe/src/main/ballerina/modules/mockextension/tracing.bal

```diff
@@ -15,14 +15,39 @@
 // under the License.
 
 import ballerina/jballerina.java;
+import ballerina/log;
+
+public type Fields record {
+    map<string> CHECKPOINT;
+};
+
+public type Event record {
+    int timestampMicros;
+    Fields fields;
+};
```

Comment:
Let's try to match the OpenTelemetry Event structure.

An event should have
- name
- attributes

Reply:
BMockEevent only has `timestampMicros` and `Fields`.